docs/run-platform-server/prerequisites.mdx

```diff
@@ -3,56 +3,41 @@ title: Prerequisites
 
-With these three functions in mind, you can also run Horizon in two different ways: **real-time ingestion** and **historical catch-up**:
+To run the Horizon service with all three functions on a single compute instance, the minimum hardware specifications are:
 
-- _Real-time ingestion_ is an “online” process: it involves keeping in sync with the live Stellar network state and digesting ledger data into a holistic view of the network. If you just want function (3) from above, you still need to do this.
+**For the compute instance:**
 
-- _Historical catch-up_ is an “offline” process: it lets you look into the past and catch up your Horizon instance to a given retention period (e.g. 30 days of history). Because it’s typically done offline and a one-time process, you can dedicate more compute power and configure parallel workers to catch up faster.
+- CPU: x cores
+- RAM: x GB
+- Storage: SSD with a capacity of x GB
 
-### Historical Catch-up
+**For the PostgreSQL database instance:**
 
-In this scenario, the hardware specifications are more demanding than what is necessary for the day-to-day operation of real-time ingestion, but catch-up only needs to occur once.
+- CPU: x cores
+- RAM: x GB
+- Storage: SSD with a capacity of x TB (NVMe or Direct Attached Storage) capable of handling >xk IOPS (I/O operations per second).
 
-However, the requirements will vary depending on your chosen retention period and desired catch-up speed. Note that **most operators will not need full history**, and as the network continues to grow, tracking full history will become increasingly prohibitive. As of late 2021, DB storage to support historical retention is growing at a rate of 0.8 TB / month. It is highly recommended to configure retention of only the history needed to support your functionality.
+These specifications consider a 30-day retention window for data storage. For a longer retention window, the system requirements will change accordingly. Consult the [ingestion](https://developers.stellar.org/docs/run-api-server/ingestion) for details on data ingestion and history retention, and managing storage requirements.
 
-#### Requirements
+**Network**
 
-Minimally, your disk storage type **must** be an SSD (e.g. NVMe, Direct Attached Storage) and your I/O **must** handle >15k iops (I/O operations per second). The following table breaks down hardware specifications for ingestion at different retention levels and performance tiers.
+- The compute instance must be able to connect to the PostgreSQL database. The default port for PostgreSQL is 5432.
+- A stable and fast Internet connection is recommended to ensure efficient communication with the Stellar network.
 
-Note that each component can be scaled independently and for redundancy, in the manner of traditional _n_-tier systems which is covered later in [Scaling](./scaling.mdx). Ingestion can be sped up via configuring more Captive Core parallel workers (requiring more compute and RAM).
+**Production Setup**
```

Comment:
Maybe mention the production focus in general as part of the pre-amble at top and re-label this as Multi-Instance?

Reply:
done.